libs/shared/src/utils/env.ts

```diff
@@ -0,0 +1,51 @@
+type CloudflareEnv = { env: Record<string, string> };
```

Comment:
This is more of a preview for now in terms of how things should be approached in the new JS isomorphic runtime world. The package works in Node for now so process.env will work.